mirrulations-client/src/mirrclient/disk_saver.py

```diff
@@ -70,3 +70,42 @@ def is_duplicate(self, existing, new):
     def check_for_duplicates(self, path, data, i):
         if self.is_duplicate(self.open_json_file(path), data) is False:
             self.save_duplicate_json(path, data, i)
+
+    def get_extracted_text_attachments_dir(self, path):
```

Comment:
Is there already something in the path generator for this? At the very least, this should probably be in the path generator.